### ISSUE TITLE
Wrong `offset`

### ISSUE THREAD

**Issue by fisker** [postcss/postcss-scss#146]
```js
> require('postcss-scss').parse('a{//c\n}').nodes[0].nodes[0].source
{
  input: Input {
    css: 'a{//c\n}',
    hasBOM: false,
    id: '<input css aBxZx8>',
    [Symbol(fromOffsetCache)]: [ 0, 6 ]
  },
  start: { column: 3, line: 1, offset: 2 },
  end: { column: 5, line: 1, offset: 4 }
}
```

The `end.offset` should be `5`. Same as https://github.com/postcss/postcss/pull/1879 ?

**Comment by fisker** [comment]
I'm not familiar with the codebase, maybe @idoros can help?

**Comment by idoros** [comment]
I don't mind sending a PR to fix this if it helps

**Comment by ai** [comment]
Thanks to @idoros. The fix was released at 4.0.8.

**Comment by fisker** [comment]
Thanks!
